src/main/webapp/script.js

```diff
@@ -19,10 +20,17 @@ async function displayCodes() {
             let row = tableEl.insertRow(-1);
             var videoUrl = row.insertCell(0);
             var promoCode = row.insertCell(1);
+            var descriptionSnippet = row.insertCell(2);
             videoUrl.innerHTML = '<a href="https://www.youtube.com/watch?v='
                + codes[i].videoId +'" target="_blank">'
                + codes[i].videoTitle + '</a>';
             promoCode.innerHTML = codes[i].promoCode;
+            descriptionSnippet.innerHTML = boldString(codes[i].snippet, codes[i].promoCode);
         }
     }
 }
+
+function boldString(str, substr) {
```

Comment:
Change name to `boldSubstring()` to follow parameter naming since `substr` is getting bolded, not `str`.